modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -11,15 +11,13 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support `Service` objects with type `NodePort`.
+Using {product-title} with Kuryr SDN has several limitations that apply to all versions and environments:
 
-{product-title} with Kuryr SDN and OVN Octavia provider supports `Service` objects
-without specifying `.spec.selector` only when the `Endpoints` object `.subsets.addresses`
-are either on the Nodes or Pods subnet.
+* `Service` objects with the `NodePort` type are not supported.
 
-If the machines subnet is not connected to a router, or if the
-subnet is connected, but the router has no external gateway set,
-Kuryr cannot create floating IPs for `Service` objects with type `LoadBalancer`.
+* Clusters that use the OVN Octavia provider driver support `Service` objects for which the `.spec.selector` property is unspecified only if the `.subsets.addresses` property of the `EndPoints` object includes the nodes' or pods' subnet. 
```

Comment:
Perhaps it's better to leave `Endpoints` here as is the way defined on Kubernetes.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#endpoints-v1-core

Reply:
@MaysaMacedo As in `Endpoints`? 

```suggestion
* Clusters that use the OVN Octavia provider driver support `Service` objects for which the `.spec.selector` property is unspecified only if the `.subsets.addresses` property of the `Endpoints` object includes the nodes' or pods' subnet. 
```

Reply:
@MaysaMacedo ^^^

Reply:
Yep, it should not be camelcased.